### PR TITLE
fix: Removing items during validation breaks validation

### LINKED ISSUE #266: ValueError: Uncached item of id $id in media/cache.py
**Describe the bug**
```
botamusique[2463]: [May 04 18:16:22 INFO] bot: play music [url] “Someone in the Crowd” - La La Land (2016) (https://www.youtube.com/watch?v=ocHKTdv-l9U) from playlist La La Land (2016) - F>
botamusique[2463]: Traceback (most recent call last):
botamusique[2463]:   File "/nix/store/slcm3hnfpv03pyvvqd7rrci1imp52vvm-botamusique-unstable-2021-03-13/share/botamusique/.mumbleBot.py-wrapped", line 896, in <module>
botamusique[2463]:     var.bot.loop()
botamusique[2463]:   File "/nix/store/slcm3hnfpv03pyvvqd7rrci1imp52vvm-botamusique-unstable-2021-03-13/share/botamusique/.mumbleBot.py-wrapped", line 560, in loop
botamusique[2463]:     self.async_download_next()
botamusique[2463]:   File "/nix/store/slcm3hnfpv03pyvvqd7rrci1imp52vvm-botamusique-unstable-2021-03-13/share/botamusique/.mumbleBot.py-wrapped", line 415, in async_download_next
botamusique[2463]:     if not next.is_ready():
botamusique[2463]:   File "/nix/store/slcm3hnfpv03pyvvqd7rrci1imp52vvm-botamusique-unstable-2021-03-13/share/botamusique/media/cache.py", line 188, in is_ready
botamusique[2463]:     return self.item().is_ready()
botamusique[2463]:   File "/nix/store/slcm3hnfpv03pyvvqd7rrci1imp52vvm-botamusique-unstable-2021-03-13/share/botamusique/media/cache.py", line 145, in item
botamusique[2463]:     raise ValueError(f"Uncached item of id {self.id}.")
botamusique[2463]: ValueError: Uncached item of id 40d56eeac2a7a5b53af4506829a2e297.
systemd[1]: botamusique.service: Main process exited, code=exited, status=1/FAILURE
```

**Affected version**
df38c7dbd6d59c6790cf2364d1f344b7f6f72107

**To Reproduce**
Unclear, I wasn't using the bot at the time, the users said a song finished properly and the bot just quit the mumble session.

**Expected behavior**
Don't crash on caching issues.

**Additional context**
https://github.com/azlux/botamusique/blob/master/media/cache.py#L145

### PR DESCRIPTION
**Pull request by TerryGeng**
Fix #266.

### CODE REVIEW

**Comment by TerryGeng** [comment]
@mweinelt Would you help me check if this quick fix solved your issue?

File: media/cache.py [hunk 1]
```diff
@@ -142,7 +146,7 @@ def item(self):
         if self.id in self.lib:
             return self.lib[self.id]
         else:
```
Comment (mweinelt) [inline media/cache.py:148]: Everything after the return is automatically an else block.

Comment (TerryGeng) [reply media/cache.py:148]: But it does no harm to explicitly use an else statement :)

**Comment by mweinelt** [comment]
Haven't seen the issue again so far.

**Comment by TerryGeng** [comment]
Great! Thanks for testing.

**Pull request merged**